src/lib/cache/redis/redis.go

```diff
@@ -107,23 +91,45 @@ func New(opts cache.Options) (cache.Cache, error) {
 		opts.Address = "redis://localhost:6379/0"
 	}
 
-	name := fmt.Sprintf("%x", sha256.Sum256([]byte(opts.Address)))
-
-	param := &libredis.PoolParam{
-		PoolMaxIdle:           100,
-		PoolMaxActive:         1000,
-		PoolIdleTimeout:       10 * time.Minute,
-		DialConnectionTimeout: time.Second,
-		DialReadTimeout:       time.Second * 2,
-		DialWriteTimeout:      time.Second * 5,
+	u, err := url.Parse(opts.Address)
```

Comment:
please comment the defaults values in go-redis.

Reply:
updated.